src/Files.App/App.xaml.cs

```diff
@@ -174,8 +176,9 @@ private static async Task StartAppCenter()
                 {
                     var file = await StorageFile.GetFileFromApplicationUriAsync(new Uri(@"ms-appx:///Resources/AppCenterKey.txt"));
                     var lines = await FileIO.ReadTextAsync(file);
-                    var obj = Newtonsoft.Json.Linq.JObject.Parse(lines);
-                    AppCenter.Start((string)obj.SelectToken("key"), typeof(Analytics), typeof(Crashes));
+                    using var document = System.Text.Json.JsonDocument.Parse(lines);
+                    var obj = document.RootElement;
+                    AppCenter.Start(obj.GetPropertyValue<string>("key"), typeof(Analytics), typeof(Crashes));
```

Comment:
It appears that this isn't reading the json file correctly.

Reply:
You can test by adding data and seeing that it doesn't work (pipeline adds data when the app is built)

Reply:
This should fix the problem
`AppCenter.Start(obj.GetProperty("key").GetString(), typeof(Analytics), typeof(Crashes));`
Should I open a new PR?

Reply:
> Should I open a new PR?

Please 👍 
Can do the same for the bing api key